src/app/design/styles/typography/index.html

```diff
@@ -27,7 +31,7 @@ <h1 class="sky-page-heading" id="page-heading">Page heading</h1>
       </div>
       <div class="definition-styling">
         <div class="sky-field-label">Styling</div>
-        <div>Blackbaud Sans Condensed Light 34px #212327</div>
+        <div><a href="https://sky.blackbaudcdn.net/skyux-fonts/1.0.1/blackbaud-sans-condensed-light.woff">Blackbaud Sans Condensed Light 34px #212327</a></div>
```

Comment:
I would vote to update the links to just cover the font name, and not the other properties.